CriticalMass/BikeAnnotationView.swift

```diff
@@ -40,17 +45,18 @@ class FriendAnnotationView: MKAnnotationView {
         super.init(coder: aDecoder)
         commonInit()
     }
+
     private func commonInit() {
         canShowCallout = false
 
-        let friendView = FriendVie(frame: .init(x: 0, y: 0, width: 109, height: 23))
-        friendView.backgroundColor = .clear
-        friendView.name = friend?.name ?? ""
-        addSubview(friendView)
+        friendView = FriendView(frame: .init(x: 0, y: 0, width: 109, height: 23))
+        friendView!.backgroundColor = .clear
+        friendView!.name = friend?.name ?? ""
+        addSubview(friendView!)
```

Comment:
Maybe we judt make the FriendView implicitly unwrapped so we can loose a couple of❗️